client/pom.xml

```diff
@@ -384,23 +408,6 @@
         </executions>
       </plugin>
       <plugin>
-        <groupId>com.googlecode.todomap</groupId>
-        <artifactId>maven-jettygzip-plugin</artifactId>
-        <version>0.0.4</version>
-        <configuration>
-          <webappDirectory>${project.build.directory}/generated-webapp</webappDirectory>
-          <outputDirectory>${project.build.directory}/generated-webapp</outputDirectory>
-        </configuration>
-        <executions>
-          <execution>
-            <phase>prepare-package</phase>
-            <goals>
-              <goal>process</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
```

Comment:
I assume you want to configure dynamic configure dynamic compression in httpd, but that will take cpu-time each time you load the page and therefore it may or may not save time.